src/main/resources/explanations/challenge34_hint.adoc

```diff
@@ -0,0 +1,6 @@
+Like other Git challenges this can be solved by manually combing the Git metadata. 
+There are also a plethora of tools that will automatically search for secrets leaked in Git repos.
+
+1. Search manually using `git log`. You can then use a tool such as grep to try and find leaked secrets with regex searches.
+
+2. Search a whole Git repo with a secrets scanning tool such as TruffleHog.
```

Comment:
If the tool does not pick it up, then let's not recommend it. Instead: can we have a look at alternatives? e.g. is there a way to see the notes here in github?